src/components/organisms/UiMultipleChoices/UiMultipleChoices.vue

```diff
@@ -151,20 +163,15 @@ const setFirstMultipleAnswerItemRef = (
   const multipleChoicesItem = el as InstanceType<typeof UiMultipleChoicesItem>;
 
   if (multipleChoicesItem.content && multipleChoicesItem.content[0].content && elHasError) {
-    invalidMultipleChoicesItemRefs.set(idx, multipleChoicesItem.content[0].content.input);
+    invalidInputsRefs.set(idx, multipleChoicesItem.content[0].content.input);
   }
 };
 
-watch(
-  invalidMultipleChoicesItemRefs,
-  (val) => {
-    if (val.size > 0) {
-      const element = [ ...val.values() ][0];
-      focusElement(element, true);
-    }
-  },
-  { immediate: true },
-);
+defineExpose<ExposedTypes>({
+  invalidInputs: invalidInputsRefs,
+  focusInvalidChoice,
```

Comment:
I think we only ned to expose `focusInvalidChoice`. I don't see how `invalidInputs` could be useful.